packages/amplify-e2e-tests/src/__tests__/geo-headless.test.ts

```diff
@@ -40,21 +40,34 @@ describe('Geo headless tests', () => {
           setAsDefault: true
         }
       };
+      const removeGeoRequest: RemoveGeoRequest = {
+        version: 1,
+        serviceRemoval: {
+          serviceName: "Map",
+          name: mapId
+        }
+      };
       await initJSProjectWithProfile(projRoot, {});
       await addAuthWithDefault(projRoot);
+      //add map
       await addHeadlessGeo(projRoot, addGeoRequest);
       await amplifyPushAuth(projRoot);
-
-      const meta = getProjectMeta(projRoot);
+      let meta = getProjectMeta(projRoot);
       const { Name: name, Region: region } = Object.keys(meta.geo).map(key => meta.geo[key])[0].output;
       expect(name).toBeDefined();
       expect(region).toBeDefined();
       const { MapName: mapName } = await getMap(name, region);
       expect(mapName).toBeDefined();
+      //update map
       await updateHeadlessGeo(projRoot, updateGeoRequest);
       await amplifyPushAuth(projRoot);
-      const newMeta = getProjectMeta(projRoot);
-      expect(newMeta.geo[mapId].accessType).toBe('AuthorizedAndGuestUsers');
+      meta = getProjectMeta(projRoot);
+      expect(meta.geo[mapId].accessType).toBe('AuthorizedAndGuestUsers');
+      //remove map
+      await removeHeadlessGeo(projRoot, removeGeoRequest);
+      await amplifyPushAuth(projRoot);
+      meta = getProjectMeta(projRoot);
+      expect(meta.geo[mapId]).toBeUndefined();
```

Comment:
Can you try to add 2 Maps and remove the default map to check if the second will be set to default? That would give completeness to this test. Thanks.